spec/lib/sufia/export/collection_converter_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe Sufia::Export::CollectionConverter do
+  let(:collection) do
+    Collection.create(title: "title1", creator: ["creator1"], description: "description1") do |col|
```

Comment:
@mjgiarlo This is sufia 6, so there are no factories in existence. Also for Sufia 6 title and description are singular.